source/daplink/drag-n-drop/flash_manager.c

```diff
@@ -237,6 +238,11 @@ error_t flash_manager_uninit(void)
     return ERROR_SUCCESS;
 }
 
+void flash_manager_set_page_erase(bool enabled)
+{
+    page_erase_enabled = enabled;
```

Comment:
Might want to assert that the state is STATE_CLOSED since this could have unintended side effects if changed in the middle of programming.